community/meetup/README.md

```diff
@@ -0,0 +1,31 @@
+Welcome to the Knative Community Meetup page!
+
+The virtual event is designed for end users, a space for our community to meet, get to know each other, and learn about uses and applications of Knative.
+
+Here we will list all the information related to past and future events.
+
+Stay tuned for new events by subscribing to our calendar and following the [@KnativeProject](https://twitter.com/KnativeProject) on Twitter.
```

Comment:
A link to the said calendar (or a page that contains a link to it) would be helpful.

Reply:
hey there!

so, yeah, this is the point `1`from the PR description: I think it would be best to have a calendar with only the Knative Community events.

I'm gonna drop this request in the #community-meetup-planning channel on Slack :)

Reply:
My bad, I though you were referring to the existing Knative calendar.

Reply:
no worries mate!

Reply:
Added the link :)